sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,246 @@
+ - Fixed a problem which could lead to incorrect user header to be picked up
+   [c23fe4b]
+### SYCL Library
+ - Added assign operator to specializations of `sycl::ext::oneapi::atomic_ref`
+   [c6bc5a6]
+ - Fixed the way managed memory is freed in CUDA plugin [e825916]
+ - Changed names of  some SYCL internal enumerations to avoid possible
+   conflicts with user macro [1419415]
+ - Fixed status which was returned for host events by
+   `event::get_info<info::event::command_execution_status>()` call [09715f6]
+ - Fixed memory ordering used for barriers [73455a1]
+ - Fixed several places in CUDA and HIP plugins where `bool` was used instead
+   of `uint32_t` [764b6ff]
+ - Fixed event pool memory leak in Level Zero plugin [0e95e5a]
+ - Removed redundant `memcpy` call for copying struct using `fpga_reg`
+   [a5d290d5]
+ - Fixed an issue where the native memory object passed to interoperability
+   memory object constructor was ignored on devices without host unified memory
+   [da196784a]
+ - Fixed a bug in `simd::replicate_w` API [d36480da]
+ - Fixed group operations for `(u)int8/16` types [6a055ec2]
+ - Fixed a problem with non-native specialization constants being undefined if
+   they are not explicitly updated to non-default values [3d96e1d]
+ - Fixed a crash which could happen when a default constructed event is passed
+   to `sycl::handler::depends_on`[2fe7dd3]
+ - Fixed `sycl::link` which was returning a kernel bundle in the incorrect state
+   [6d98beb]
+ - Fixed missing dependency for host tasks in in-order queues [739487c]
+ - Fixed a memory leak in Level Zero plugin [a5b221f]
+ - Fixed a problem with copying buffers when an offset is specified in CUDA
+   backend [0abdfd6]
+ - Fixed a problem which was preventing passing asynchronous exceptions produced
+   in host tasks [f823d61]
+ - Fixed a crash which could happen when submitting tasks to multiple queues
+   from multiple threads [4ccfd21]
+ - Fixed a possible multiple definitions problem which could happen when using
+   `fsycl-device-code-split` option [d21082f]
+ - Fixed several problem in configuration file processing [2a35df0]
+ - Fixed imbalance in events release and retain in Level Zero plugin [6117a1b]
+
+## API/ABI breakages
+
+## Known issues
+ - [new] SYCL 2020 barriers show worse performance than SYCL 1.2.1 do [18c80fa]
+ - [new] When using fallback assert in separate compilation flow requires
+   explicit, linking against `lib/libsycl-fallback-cassert.o` or
+   `lib/libsycl-fallback-cassert.spv`
+ - [new] Performance may be impacted by JIT-ing an extra 'copier' kernel and due
+   running the 'copier' kernel and host-task after each kernel which uses
+   assert
+ - [new] Driver issue. When a two-step AOT build is used and there's at least a
+   single call to devicelib function from within kernel, the device binary
+   image gets corrupted
+ - [new] Filenames are removed from diagnostic to avoid leaks of full paths
+   from build environment [6bbfe42b3]
```

Comment:
I don't think it's an issue. It's a change and it's a fix.

Reply:
Moved to bug fixes section.